### DISCUSSION TITLE
SyntaxError when trying to add new subscription.

### DISCUSSION THREAD

**Discussion by mrtargaryen** [ellite/Wallos#387] [General]
Wallos v2.4.2
Docker 26.1.3

When I try to add a new subscription, I receive the following error:

`SyntaxError: JSON.parse: unexpected end of data at line 1 column 1 of the JSON data`

I don't know what this means nor how to resolve it. I did a Google search for information, but I didn't understand anything I read.

Could anyone point me in the right direction?

**Comment by ellite** [comment]
If you could open the developer tools on the browser, open the **network** tab and try to add the subscription again.
Click on the failed `add.php` and go the **response** tab. It should give more information on what the error actually is.
Then please paste the error message here.

Thanks.

**Reply by mrtargaryen** [reply]
I did as you said, but for some reason it just worked when I added the subscription, so it must have just been a small glitch. Thanks for the advice, though.

**Comment by jlar0che** [comment]
Had the same exact issue. Opened a new tab, tried again and it worked. There is some elusive underlying issue here...
